Sources/XiEditor/Search.swift

```diff
@@ -543,6 +543,9 @@ extension EditViewController {
 
         case .hideReplaceInterface:
             Swift.print("hideReplaceInterface not implemented")
+
+        @unknown default:
```

Comment:
what's this syntax? Can you link me a reference?

Reply:
I just learned about it while upgrading also. It allows us to have an exhaustive enum with a `default` case to protect against future additions to the enum.  If we don't put the `@unknown` on the `default`, the compiler will complain that we've covered all the cases already.

Reply:
cool, thanks!